lib/serviceAccessConfig/nginx.py

```diff
@@ -0,0 +1,83 @@
+#  Copyright (C) 2019 SUSE LLC
+#  All rights reserved.
+#
+#  This file is part of serviceAccessConfig
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""serviceAccessConfig plugin for nginx"""
+
+import ConfigParser
+import logging
+
+from accessrulegenerator import ServiceAccessGenerator
+from generatorexceptions import *
```

Comment:
Is any import used from this module? It should be imported explicitly if so instead of wildcard.